### PR TITLE
add achievement doc

### PR DESCRIPTION
**Pull request by ParvathiKrishnan99**

### CODE REVIEW

File: sidebar.json [hunk 1]
```diff
@@ -154,6 +154,10 @@
       {
         "name": "Favorite Languages",
         "slug": "favorite-languages"
+      },
+      {
+        "name": "Earn Achievements",
```
Comment (jeremypress) [inline sidebar.json:159]: I think we should leave this unlisted for now, so only explorers can get to it

File: misc/achievements.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Earning Achievements [BETA]
+
+###### Note that achievements is currently only enabled for explorers.  Learn to become an explorer [here](https://repl.it/talk/announcements/Become-an-Explorer/6180)
+
+Achievements is a new Repl.it feature rolling out in the summer 2020 for some users. These achievements are given out when a user completes a certain number of actions or a specific action. Achievements is a way to encourage replers to navigate the platform, engage with the community and meet personal goals.
```
Comment (jeremypress) [inline misc/achievements.md:5]: I would skip "some users"

can you check with the team if we refer to users as "replers"? Haven't seen that in a production context.

File: misc/achievements.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+<img src="../static/images/achievements/mask-off.png" style="width: 50px;" />
+
+**Mask Off:** Earned when you update your bio and profile picture on your Repl.it Account
+
+* * *
+
+<img src="../static/images/achievements/contributor.png" style="width: 50px;" />
+
+**Repl.it Contributor:** Earned when you actively submit new posts on Repl Talk
```
Comment (jeremypress) [inline misc/achievements.md:19]: I think you can skip "actively" here

File: misc/achievements.md (hunk 2 above)
Comment (ParvathiKrishnan99) [reply misc/achievements.md:5]: Amjad mentioned it in the initial achievements docs but I'll clarify with the team.

**Pull request merged**